src/ngraph/frontend/onnx_import/utils/common.cpp

```diff
@@ -46,6 +46,41 @@ namespace ngraph
                                        static_cast<onnx::TensorProto_DataType>(onnx_type)));
             }
 
+            std::size_t validate_axis(const ngraph::onnx_import::Node& node,
+                                      std::int64_t axis,
+                                      std::size_t tensor_rank)
+            {
+                std::int64_t new_axis = axis;
+
+                if (new_axis < 0)
```

Comment:
You are not really checking any `new` axis here, but the provided axis value. Using the name `new_axis` is misleading in this check.